src/models/modules/bricks/mlp.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+import torch.nn as nn
+from torch import Tensor
+
+
+class Mlp(nn.Module):
```

Comment:
Depending on whether Swin gets merged first, you should take one of the two implementations